README.md

```diff
@@ -24,4 +23,10 @@ This plugin allows you to censor profanity on your Mattermost server. The plugin
 
 ### Usage
 
-You can edit the bad words list in **System Console > Plugins > Profanity Filter > Bad Words list**.
+You can edit the bad words list in **System Console > Plugins > Profanity Filter > Bad words list**.
+
+Choose to either censor the bad words with a character or reject the post with a custom warning message:
+
+![Post rejected by the plugin](./.github/post-rejected.gif)
```

Comment:
`.github` is usually used for config files. Other plugins use an `images` directory. Mind moving the files?

Reply:
Of course!